docs/source/tutorials/configure_systems.rst

```diff
@@ -7,6 +7,19 @@ A system is the environment (OS, hardware spec, device platform, supported EP) t
 is evaluated on. It can thus be the **host** of a Pass or the **target** of an evaluation. This document describes
 how to configure the different types of Systems.
 
+Accelerator Configuration
+-------------------------
+For each **host** or **target**, it could specify multiple accelerators associated with it. Each accelerator could have the following attributes.
+
+* :code:`device`: The device type of the accelerator. It could be "cpu", "gpu", "npu", etc. Please refer to the API documentation for the full list of supported devices.
+* :code:`execution_providers`: The execution provider list that are supported by the accelerator. For e.g. ["CUDAExecutionProvider", "CPUExecutionProvider"].
+
+Note:
+
+- The accelerators for local system or python system is optional. If not provided, Olive will get the available execution providers installed in current local machine and infer its device.
+- If the accelerators are specified, the device is mandatory. The execution_providers is optional. If not provided, Olive will get the available execution providers associated with the device.
```

Comment:
from the PR description and this, it looks like the inferring device from ep is now removed? I think that was a useful feature since some of the information is redundant. if the ep is cuda, the device is always gpu. same for qnn ep and npu.

Reply:
also what is the difference between 
`[{device: gpu, ep=[cpuep, cudaep]}]` and `[{device:gpu, ep:[cpuep]}, {device:gpu, ep:[cudaep]}]`

Reply:
Actually, I am not sure whether the infer device from EP is required feature after both device and ep is consolidated. From my understanding, I think ep is specified but device not is a little bit confusing. 

@devang-ml, how do you think?
@jambayk, for your second question, they are same after flatten to AcceleratorSpec.

Reply:
I have same question here. I can't remember the reason why we have both device and ep, but seems only one of them is needed. If user specified device, we can use all available eps for this device. If user specified eps, we can run Olive with those eps.

Reply:
A generic EP could support GPU and CPU both so we need device as well as EP.
In cases where there is a clear mapping between EP and the device (e.g. CudaEP) then we can make device optional as a convenience.

Reply:
updated.